### PR TITLE
doc: update README

### PR DESCRIPTION
**Pull request by Burning1020**
Add supported plan of sandboxes

### CODE REVIEW

**Review by abel-von — COMMENTED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -18,6 +18,20 @@
...
+|----------------|------------------|--------------------|
+| **MicroVM**    | Cloud Hypervisor | Supported          |
+|                | QEMU             | Supported          |
+|                | StratoVirt       | Supported          |
+|                | Firecracker      | Planned in 2024    |
+| **App Kernel** | Quark            | Supported          |
+|                | gVisor           | Planned in 2024    |
+| **Wasm**       | WasmEdge         | Supported          |
+|                | Wasmtime         | Supported          |
+| **runC**       | runC             | Planned in 2023 H2 |
```
Comment (kevin-wangzefeng) [inline README.md:33]:
```suggestion
| Sandboxer      | Sandbox          | Status             |
|----------------|------------------|--------------------|
| **MicroVM**    | Cloud Hypervisor | Supported          |
| **MicroVM**    | QEMU             | Supported          |
| **MicroVM**    | StratoVirt       | Supported          |
| **MicroVM**    | Firecracker      | Planned in 2024    |
| **App Kernel** | Quark            | Supported          |
| **App Kernel** | gVisor           | Planned in 2024    |
| **Wasm**       | WasmEdge         | Supported          |
| **Wasm**       | Wasmtime         | Supported          |
| **runC**       | runC             | Planned in 2023 H2 |
```

**Review by kevin-wangzefeng — COMMENTED** [review]
And I'd suggest include plan of sandboxers that support confidential computing

**Review by kevin-wangzefeng — COMMENTED** [review]
And I'd suggest include plan of sandboxers that support confidential computing

**Comment by kevin-wangzefeng** [comment]
LGTM

**Pull request merged**